generators/app/index.js

```diff
@@ -231,12 +237,28 @@ module.exports = generators.Base.extend({
       this.destinationPath('.babelrc')
     );
 
-    ['gulpfile.js', 'client', 'config', 'server', 'test'].forEach((f) => {
+    ['gulpfile.js', 'config', 'server', 'test'].forEach((f) => {
       this.fs.copy(
         this.templatePath(f),
         this.destinationPath(f)
       );
     });
+
```

Comment:
odd, it works for me. it may be require an actual string.  `!this.props.pwa && '**/client/sw-register.js' || ''`